packages/gatsby/src/redux/actions/add-page-dependency.ts

```diff
@@ -0,0 +1,40 @@
+import { store } from "../"
+
+import { createPageDependency as internalCreatePageDependency } from "./internal"
+
+export const createPageDependency = ({
+  path,
+  nodeId,
+  connection,
+}: Parameters<typeof internalCreatePageDependency>[0]): void => {
+  const {
+    componentDataDependencies: { connections, nodes },
+  } = store.getState()
```

Comment:
lets also revert this destructuring. It's easier to grok the code below when you see the object path it follows.